Prelude/Operators.swift

```diff
@@ -1,41 +1,54 @@
+precedencegroup LeftApplyPrecendence {
+  associativity: left
+  higherThan: AssignmentPrecedence
+  lowerThan: TernaryPrecedence
+}
+
+precedencegroup FunctionCompositionPrecedence {
+  associativity: left
```

Comment:
well it's both! cause `(f • g) • h = f • (g • h)`. but maybe using right plays nicer with something else? will make the switch